### PR TITLE
Add brightness slider for Lights.

### PR DESCRIPTION
**Pull request by RoboMagus**
Adds a brightness slider for lights that support it.

I've purposely not touched upon RGB, or color temperature as that seems to require additional data and probably other UI elements than what can be acomplished using sliders.

Should fix issue #15.

Example of added sliders:
image

### CODE REVIEW

File: v2/esp-entity-table.ts [hunk 1]
```diff
@@ -316,7 +331,7 @@ export class EntityTable extends LitElement {
           background-color: var(--primary-color, currentColor);
         }
         input[type="range"] {
-          width: calc(100% - 8rem);
+          width: calc(100% - 4rem);
```
Comment (anatoly-savchenkov) [inline v2/esp-entity-table.ts:334]: Please revert this change. You are not taking into account the fact that labels can have more than 4 symbols ('0' on the left and '255' on the right). Climate can have more characters in labels, see https://github.com/esphome/issues/issues/3479

Comment (RoboMagus) [reply v2/esp-entity-table.ts]: I did not take that into account. I have reverted this specific change such that the rest of the PR can continue.
The main reason I had incorporated this change is that on mobile devices the slider could become more narrow than a toggle switch, making the whole thing quite finicky to use.
It's a separate issue though so I'll see what can be done about that in a separate thread.

**Comment by cociweb** [comment]
@jesserockz What's your opinion?

**Review by jesserockz — APPROVED** [review]
